cpp/src/io/orc/stripe_enc.cu

```diff
@@ -1135,8 +1140,8 @@ __global__ void __launch_bounds__(256)
     uint32_t blk_size = min(comp_blk_size, ss.stream_size - min(b * comp_blk_size, ss.stream_size));
     blk_in->srcDevice = src + b * comp_blk_size;
     blk_in->srcSize   = blk_size;
-    blk_in->dstDevice = dst + b * (3 + comp_blk_size) + 3;  // reserve 3 bytes for block header
-    blk_in->dstSize   = blk_size;
+    blk_in->dstDevice = dst + b * (3 + max_comp_blk_size) + 3;  // reserve 3 bytes for block header
+    blk_in->dstSize   = max_comp_blk_size;
```

Comment:
Make the 3 a constexpr somewhere.  I see some other random 3's roaming around :)